src/python/kiss_icp/datasets/rosbag.py

```diff
@@ -69,12 +69,13 @@ def read_point_cloud(self, bagfile: Path, topic: str, idx: int):
 
         t_field = None
         for field in msg.fields:
-            if field.name == "t" or field.name == "timestamp":
+            if field.name == "t" or field.name == "timestamp" or field.name == "time":
```

Comment:
Would you mind improving this a bit? :
```py
if field.name in ["t", "timestamp", "time"]:
```

Reply:
Done c297133565d7aeb557b1a9d549ded93747722ef1